ideascube/migrations/0009_add_a_system_user.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from ideascube.models import User
+
+
+def add_user(*args):
+    User(serial='__system__', full_name='System', password='!!').save()
```

Comment:
The fact that it is stored like this means that no hashed password will ever match this `!!` value.